fahrschein/build.gradle

```diff
@@ -0,0 +1,22 @@
+plugins {
+    id 'fahrschein.java-conventions'
+    id 'fahrschein.maven-publishing-conventions'
+}
+
+dependencies {
+    api project(':fahrschein-http-api')
+    api project(':fahrschein-http-simple')
+    api "com.fasterxml.jackson.core:jackson-databind:${property('jackson.version')}"
+    implementation "com.fasterxml.jackson.module:jackson-module-parameter-names:${property('jackson.version')}"
```

Comment:
If fahrschein explicitly uses these Jackson modules, I think, there is no other was to add it to the implementation detail, as otherwise the library user would need to make sure that these are available as a runtime dependency.

Jackson is able to load additional modules via `ServiceLoader`, but this would only work if fahrschein would only use the modules when they are there and fallback to some defaults when they are missing.

Reply:
Or were you also referring to the core Jackson libraries being used?

Reply:
background read: https://docs.gradle.org/current/userguide/java_library_plugin.html#sec:java_library_separation

So, for example 'jackson-module-parameter-names' will be part of the runtime classpath (therefore usable by fahrschein) but projects depending on fahrschein will not be able to use it in their compilation unless explicitly called as direct dependency.